docs/_static/style.css

```diff
@@ -300,6 +300,24 @@ header > nav > a:hover {
 
 .sub-header > select {
   display: none;
+  background-color: transparent;
+  padding: 0.4em 0;
+  font-size: 1em;
+  width: 13.5%;
+  color: white;
+  border: none;
+  border-bottom: 1px solid var(--search-border);
+
+  appearance: none;
+  background-image: url("data:image/svg+xml;utf8,<svg fill='white' height='24' viewBox='0 0 24 24' width='24' xmlns='http://www.w3.org/2000/svg'><path d='M7 10l5 5 5-5z'/><path d='M0 0h24v24H0z' fill='none'/></svg>");
```

Comment:
Please use an external file to load your SVGs.

Reply:
Fixed that. I didn't get to test it so let me know if I missed anything.